src/Devices/EEPROM_TC.h

```diff
@@ -26,16 +26,20 @@ class EEPROM_TC {
   void getMac(uint8_t* bytes);        // used for DHCP request for IP address
   float getMaxDataAge();              // not used
   float getPH();                      // target pH
-  uint32_t getRampTimeStart();
-  uint32_t getRampTimeEnd();
+  uint32_t getPhRampTimeStart();
+  uint32_t getPhRampTimeEnd();
   float getRampStartingPH();
+  uint32_t getPhSineStartTime();
+  uint32_t getPhSinePeriod();
+  float getPhSineAmplitude();
   uint32_t getRampTimeStartTemp();
   uint32_t getRampTimeEndTemp();
   float getRampStartingTemp();
-  float getPHDelay();            // not used
-  float getPHInterval();         // not used
-  float getPHSeriesPointer();    // not used
-  float getPHSeriesSize();       // not used
+  float getPHDelay();          // not used
```

Comment:
"Ph" or "PH"? Lets be consistent and make it always "Ph" in the function names. Update for the whole file.